### DISCUSSION TITLE
Please make the Pull Request UI more like Azure DevOps

### DISCUSSION THREAD

**Discussion by TheMagnificent11** [community/community#14867] [Pull Requests] [answered]
Everything about GitHub is better than Azure DevOps, except PRs...in my opinion.

Can you please make the pull request user experience more like the pull request user experience in Azure DevOps?

The file navigator in Azure DevOps is much better.  I like how it only shows the selected file in the right pane of the split pane view.

The auto-merge option when checks complete is my favourite PR feature in Azure DevOps...sadly missing in GitHub.

**Comment by TheMagnificent11** [comment, marked as answer]
I've just been informed that the auto-merge feature is available in the PR settings.

So I guess my issue is with the beta file explorer...never mind.

**Reply by Sigvdhdhtcychxbufv** [reply]
Apt update

**Reply by brunosquillace** [reply]
Great job mag 11 
    Proud of you

**Comment by rungrotArunon** [comment]
Thanks you

**Reply by brunosquillace** [reply]
Great job I'm proud of you
